app.rb

```diff
@@ -19,22 +12,17 @@
 # app = App.new
 # app.run
```

Comment:
- [ ] Kindly remove commented code to clean this file.

Reply:
Thank you